luigi/execution_summary.py

```diff
@@ -248,12 +255,14 @@ def _get_comments(group_tasks):
     "already_done",
     "completed",
     "failed",
+    "unknown",
     "still_pending",
     "still_pending_ext",
     "run_by_other_worker",
     "upstream_failure",
     "upstream_missing_dependency",
     "upstream_run_by_other_worker",
+    "upstream_unverifiable_dependency",
```

Comment:
This is called "unknown"/"upstream_unverifiable_dependency". That pair doesn't really match. Maybe rename to "scheduling_error"/"upstream_scheduling_error"?